CMakeLists.txt

```diff
@@ -78,6 +78,7 @@ endif ()
 
 option(VAST_ENABLE_ASSERTIONS "Enable Assertions" "${VAST_ENABLE_ASSERTIONS_DEFAULT}")
 option(VAST_RELOCATABLE_INSTALL "Enable relocatable installations" ON)
+option(VAST_NO_SYSTEM_CAF "Always use the CAF submodule" OFF)
```

Comment:
I'd recommend something `VAST_USE_BUNDLED_CAF` to avoid the double negative (same goes for `VAST_NO_ARROW`, but that's not for this PR ;)